src/ng/services/table.js

```diff
@@ -19,4 +41,13 @@ export default class Table {
 
 		return null;
 	}
+	get body(){
+		return new Dom(this.markup.body) || noop;
```

Comment:
shouldnt be noop should be something like this.markup.body ? new Dom(this.markup.body) : Dom.empty